app/views/layouts/nav/_develop_dropdown.html.erb

```diff
@@ -1,6 +1,6 @@
 <li class="dropdown" title="Develop">
   <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">
-    <i class="fa fa-code"></i><span class="hidden-sm"> Develop</span><span class="caret"></span>
+    <i class="fa fa-code"></i><span id="develop-nav-text" class="hidden-sm"> Develop</span><span class="caret"></span>
```

Comment:
If you want to do this, wouldn't `class="visible-lg-*"` be basically the same thing?

Reply:
Misread, you were hiding this at 997px which is very close to 992px.

class ="visible-md-* visible-lg-*"

Reply:
But actually instead of visible the hidden classes are better so you can keep the text visible at the smallest width.

Reply:
At the smallest width it collapses to the hamburger.

Reply:
bootstrap "md" cuts off at 992, there is overlap between 992 and 997. 
bootstrap "lg" cuts off at 1200, there is overlap between 1200 and 1210.